src/NuGet.Core/NuGet.ProjectModel/GlobalSuppressions.cs

```diff
@@ -80,3 +80,4 @@
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.ProjectModel.ProjectRestoreMetadata.GetSources(System.Collections.Generic.IList{NuGet.Configuration.PackageSource})~System.Collections.Generic.HashSet{System.String}")]
 [assembly: SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.ProjectModel.Utf8JsonStreamLockFileTargetLibraryConverter.ReadPackageDependencyList(NuGet.ProjectModel.Utf8JsonStreamReader@)~System.Collections.Generic.IList{NuGet.Packaging.Core.PackageDependency}")]
 [assembly: SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.ProjectModel.Utf8JsonReaderExtensions.ReadNumberAsString(System.Text.Json.Utf8JsonReader@)~System.String")]
+[assembly: SuppressMessage("Style", "IDE0051:Remove unused private members", Justification = "<Pending>", Scope = "member", Target = "~M:NuGet.ProjectModel.AssetsLogMessage.#ctor(NuGet.Common.LogLevel,NuGet.Common.NuGetLogCode,System.String,System.String,System.String,System.String,System.Collections.Generic.IReadOnlyList{System.String},System.Int32,System.Int32,System.Int32,System.Int32)")]
```

Comment:
Similar to another comment, why wasn't this method deleted, and if it shouldn't be deleted, can we use an inline suppression instead?

Reply:
Similar to https://github.com/NuGet/NuGet.Client/pull/6462#discussion_r2096308727